liner2-cli

```diff
@@ -5,6 +5,9 @@ DIR="$( cd "$( dirname "$0" )" && pwd )"
 name=g419-liner2-cli
 jar=`find $DIR/$name/build/libs -name "$name*-all.jar" -type f`
 
+echo $jar
+echo $@
```

Comment:
Jest to potrzebne?

Reply:
Artefakt do usunięcia, przy uruchamianiu Liner2/run_test.sh printowało informację o użytym modelu